src/popup/searchModule.js

```diff
@@ -62,6 +69,8 @@ function showNoResultFoundMessage() {
     paragraph.textContent = 'No Images Found. Please enter a different query.';
 
     removeChildNodes(sectionContentPrimary.querySelector('.row'));
+    sectionContentPrimary.querySelector('.load-more-button-wrapper').classList.add('removeLoadMore');
```

Comment:
All the required element selectors are present in `base.js` and are already imported in `searchModule.js`. Same is the case with `loadMoreButtonWrapper`, therefore, don't use querySelector to select it.

Reply:
Yeah, that was redundant. I will fix that.